src/GEXF/Gexf.jl

```diff
@@ -2,6 +2,7 @@ module GEXF
 
 using GraphIO.EzXML
 using LightGraphs
+using MetaGraphs
```

Comment:
I don't think we need this here since you're including metagraphs.jl below, which has it.